crates/polars-core/src/chunked_array/ops/take/take_single.rs

```diff
@@ -254,7 +254,7 @@ impl TakeRandom for ListChunked {
         let opt_arr = unsafe { impl_take_random_get!(self, index, LargeListArray) };
         opt_arr.map(|arr| unsafe {
             Series::from_chunks_and_dtype_unchecked(
-                self.name(),
+                "",
```

Comment:
These drive-by changes can have large performance impact on other places.

In this case it gets cheaper, so let's leave it.  But I think we should make these very clear. Maybe even in their own PR.

Reply:
This was necessary as `.get` returns a series with the name of the origin list series while `amortized_iter` returns a series with an empty name so the comparison would fail.

Hoping to get rid of this once I have the loop without `get`

Reply:
We can alway rename temporary for the comparison.